nvme-print-stdout.c

```diff
@@ -4474,8 +4486,13 @@ static void stdout_feature_show_fields(enum nvme_features_id fid,
 		stdout_lba_status_info(result);
 		break;
 	case NVME_FEAT_FID_HOST_BEHAVIOR:
-		if (buf)
+		if (buf) {
 			printf("\tHost Behavior Support: %s\n", (buf[0] & 0x1) ? "True" : "False");
```

Comment:
Pre-existing, but this looks strange. Byte/bit 0 is "Advanced Command Retry Enable (ACRE)" according to the spec?

Reply:
Another change?

Reply:
I was just wondering if you understood why the existing code is doing this. It actually looks like a regression from `nvme-cli` 1.0, which displayed the actual fields:
```
$ nvme version
nvme version 1.16
$ nvme get-feature /dev/nvme0 --feature-id 0x16 -H
get-feature:0x16 (Host Behavior), Current value:00000000
	Host Behavior Support Advanced Command Retry Enable (ACRE): False
	Host Behavior Support Extended Telemetry Data Area 4 Supported (ETDAS): False
	Host Behavior Support LBA Format Extension Enable (LBAFEE): False
```
vs.
```
$ nvme version
nvme version 2.2.1 (git 2.2.1)
libnvme version 1.2 (git 1.2)
$ nvme get-feature /dev/nvme0 --feature-id 0x16 -H
get-feature:0x16 (Host Behavior), Current value:00000000
	Host Behavior Support: False
```
I can submit a separate fix for this.

Reply:
Submitted https://github.com/linux-nvme/nvme-cli/pull/2102
You may also want to update the definition of the Host Behavior Support feature data structure to include CDFE (see https://github.com/linux-nvme/libnvme/pull/736)

Reply:
It looks like the original lines (with ACRE/ETDAS/etc) were added in df4368321500291676984ce7d31dde14154ccd87? Not sure where it was removed though.

Reply:
It looks like the `nvme-cli` version history was rebased for 2.0. So looks like that change was just lost.

Reply:
Indeed, there was a long development phase where the two branches drifted away and some things got lost in the attempt to sync those branches up again. Sorry about that.